packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -71,7 +71,7 @@ const queryBarFirstRowStyles = css({
   // to account for their height individually.
   alignItems: 'flex-start',
   gap: spacing[2],
-  paddingLeft: spacing[2],
+  paddingLeft: spacing[1],
```

Comment:
I have never understood what this 8px was doing there and it looks even weirder to me now. I tried removing it entirely then second-guessed myself and now I have it at 4px.. We could also just revert this line.